### PR TITLE
Bing back rubygem updates and fix liquid errors

### PR DESCRIPTION
**Pull request by michael-k**
Fixes okfde/codefor.de#681

### CODE REVIEW

**Comment by michael-k** [comment]
@arduina Why?  You should review stuff before you merge it.

File: _layouts/story.html [hunk 1]
```diff
@@ -24,13 +24,13 @@ <h1 class="story--header--title">
         {{ page.title }}
       </h1>
       <p class="story--header--meta"> {{ page.where }} / {{ page.when }} /
-        {% for link in page.links | limit: 1 %}
```
Comment (MorrisJobke) [inline _layouts/story.html:27]: :boom:

**Comment by arduina** [comment]
Fine. You shouldn't commit stuff before testing it, right?

**Comment by michael-k** [comment]
I can commit what ever I want as long as I don't push it to ~~master~~ gh-pages.  The tests run travis.  I haven't set up a local test environment.

**Comment by michael-k** [comment]
Although this PR has the status `merged`, it is not part of `gh-pages`. okfde/codefor.de#690 is

**Pull request merged**